packages/kotti-ui/source/kotti-card/KtCard.vue

```diff
@@ -27,19 +27,13 @@ export default defineComponent<KottiCard.PropsInternal>({
 	props: makeProps(KottiCard.propsSchema),
 	setup(props) {
 		return {
-			imageContainerClass: computed(() => ({
-				'kt-card__image-row': true,
-				'kt-card__image-row--is-bottom':
-					props.imgPosition === KottiCard.ImagePosition.BOTTOM,
-				'kt-card__image-row--is-middle':
-					props.imgPosition === KottiCard.ImagePosition.MIDDLE,
-				'kt-card__image-row--is-top':
-					props.imgPosition === KottiCard.ImagePosition.TOP,
+			cardClass: computed(() => ({
+				'kt-card': true,
+				[`kt-card--has-${props.imgPosition}-image`]: props.imgUrl,
 			})),
-			footerClass: computed(() => ({
-				'kt-card__footer': true,
-				'kt-card__footer--is-last':
-					props.imgUrl && props.imgPosition !== KottiCard.ImagePosition.BOTTOM,
+			imageRowClass: computed(() => ({
+				'kt-card__image-row': true,
+				[`kt-card__image-row--is-${props.imgPosition}`]: props.imgUrl,
```

Comment:
technically don't need `props.imgUrl` here because there's a v-if on the template anyways